{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -94,8 +94,6 @@
 # See: http://django-storages.readthedocs.io/en/latest/index.html
 INSTALLED_APPS += ['storages', ]
 
-AWS_ACCESS_KEY_ID = env('DJANGO_AWS_ACCESS_KEY_ID')
-AWS_SECRET_ACCESS_KEY = env('DJANGO_AWS_SECRET_ACCESS_KEY')
```

Comment:
This will break non-AWS deployment that use S3 to store files, e.g. on Heroku with Media files going to a S3 Bucket.

Reply:
Thanks for pointing out this. I am gonna go ahead and revert this.